content/spin/kv-store.md

```diff
@@ -85,6 +85,25 @@ $ spin new http-ts spin-key-value
 
 {{ blockEnd }}
 
+{{ startTab "Python" }}
+
+<!-- @selectiveCpy -->
+
+```bash
+# Install the py2wasm plugin
```

Comment:
Other languages on this page (rightly or wrongly) don't show installing prerequisites such as js2wasm or the templates.  Is it intentional to treat Python differently?